docs/topics/multiplatform/multiplatform-android-layout.md

```diff
@@ -0,0 +1,107 @@
+[//]: # (title: Android source set layout)
+
+The new Android source set layout was introduced in Kotlin 1.8.0 and became default in 1.9.0. Follow our guide to
+understand the key differences between the deprecated and the new layout and migrate your projects.
+
+> You don't need to implement all the suggestions. Only those that are applicable to your particular projects. 
+> 
+{type="tip"}
+
+## Check the compatibility
+
+Check your version of the Android Gradle plugin and upgrade if necessary.
+The new layout requires Android Gradle plugin 7.0 or later and is supported in Android Studio 2022.3 and later.
+
+## Rename Kotlin source sets
+
+If applicable, rename source sets in your project, following this pattern:
+
+| Previous source set layout             | New source set layout               |
+|----------------------------------------|-------------------------------------|
+| `targetName` + `AndroidSourceSet.name` | `targetName` + `AndroidVariantType` |
+
+`{AndroidSourceSet.name}` maps to `{KotlinSourceSet.name}` as follows:
+
+|             | Previous source set layout | New source set layout          |
+|-------------|----------------------------|--------------------------------|
+| main        | androidMain                | androidMain                    |
+| test        | androidTest                | android<b>Unit</b>Test         |
+| androidTest | android<b>Android</b>Test  | android<b>Instrumented</b>Test |
+
+## Move source files
+
+If applicable, move your source files to the new directories, following this pattern:
+
+| Previous source set layout                            | New source set layout                                                     |
+|-------------------------------------------------------|---------------------------------------------------------------------------|
+| The layout had additional `/kotlin` SourceDirectories | `src/{AndroidSourceSet.name}/kotlin`, `src/{KotlinSourceSet.name}/kotlin` |
```

Comment:
I'm not sure, should we show `src/{AndroidSourceSet.name}/kotlin` and suggested paces for "migration"? Because if someone migrates code to `src/main/kotlin` or `src/test/kotlin` the warning will be produced and Kotlin Plugin will suggest moving it to related `src/{KotlinSourceSet.name}/kotlin`

cc @sellmair

Reply:
And probably, we have to mention this deprecation of the 'Android Style' source directories.

Reply:
Thank you! removed the mention of `src/{AndroidSourceSet.name}/kotlin`

Reply:
I agreed here on private message!